.travis.yml

```diff
@@ -17,7 +17,7 @@ env:
     - CCACHE_TEMPDIR=/tmp/.ccache-temp
     - CCACHE_COMPRESS=1
     - BASE_OUTDIR=$TRAVIS_BUILD_DIR/out
-    - SDK_URL=https://bitcoincore.org/depends-sources/sdks
+    - SDK_URL=https://github.com/phracker/MacOSX-SDKs/releases/download/10.13
```

Comment:
From what I read in issue # 13 in that repository, we cannot download and compile against these SDKs in the cross-compiling environment currently set up for CI:

> You are expressly prohibited from separately using the Apple SDKs or attempting to run any part of the Apple Software on non-Apple-branded hardware.

_[XCode SDK Agreement](https://www.apple.com/legal/sla/docs/xcode.pdf)_

We may need to run CI for macOS builds on [Travis' macOS builders](https://docs.travis-ci.com/user/reference/osx).

Reply:
I'm not sure if this language only applies to running the final software produced by building against the SDKs, or if it prohibits compilation against the SDKs on non-Apple hardware. FWIW, Bitcoin provides version 10.11 here: https://bitcoincore.org/depends-sources/sdks/MacOSX10.11.sdk.tar.gz

Reply:
Let's switch to Bitcoin's image. I trust it more. I have looked at that agreement before Paragraph 2.2A ii. appears to allow the use of MacOS SDK's to develop applications, as it is excluded from the more strict requirements around general Apple SDK's.

Reply:
![image](https://user-images.githubusercontent.com/7529186/68093617-91a25e00-fe65-11e9-8a3d-25736e1d1612.png)

Reply:
The language is somewhat confusing, but it clearly carves out the MacOS SDK from the Apple SDK in general and allows more permissive use of the MacOS SDK. Unless I am mistaken, that is the only thing we are using.

Reply:
And... you are right. We need to switch to the Travis MacOS VM's for Mac building...

Reply:
But that can come later.

Reply:
I agree with your conclusions.